src/acl/external/AD_group/ext_ad_group_acl.cc

```diff
@@ -258,18 +259,18 @@ My_NameTranslate(wchar_t * name, int in_format, int out_format)
         }
         WIN32_COM_initialized = 1;
     }
-    hr = CoCreateInstance(&CLSID_NameTranslate,
+    hr = CoCreateInstance(CLSID_NameTranslate,
                           nullptr,
                           CLSCTX_INPROC_SERVER,
-                          &IID_IADsNameTranslate,
+                          IID_IADsNameTranslate,
                           (void **) &pNto);
     if (FAILED(hr)) {
         debug("My_NameTranslate: cannot create COM instance, ERROR: %s\n", Get_WIN32_ErrorMessage(hr));
         /* This is a fatal error */
         exit(EXIT_FAILURE);
     }
-    static const auto emtpyStr = SysAllocString(L"");
-    hr = pNto->Init(ADS_NAME_INITTYPE_GC, emtpyStr);
+    static const auto emptyString = SysAllocString(L"");
+    hr = pNto->Init(ADS_NAME_INITTYPE_GC, emptyString);
```

Comment:
Undo this renaming-only change/leftovers?

```suggestion
    static const auto emptyStr = SysAllocString(L"");
    hr = pNto->Init(ADS_NAME_INITTYPE_GC, emptyStr);
```

Same for other similar occurrences.

Reply:
This change request remains valid. I see two related places in the current PR diff.

Unfortunately, my specific suggestion for one of those places was wrong: I did not notice emtpyStr misspelling in original code and, hence, did not reproduce it in the "undo" suggestion.

Again, I do not insist on removing these out-of-scope improvements from this PR.

Reply:
This was changed in 73d3b24961; from what I can see, no "String" suffix is used for variables anymore. Resolving

Reply:
> This was changed in [73d3b24](https://github.com/squid-cache/squid/commit/73d3b24961053f5f8f91639c2d8706b93886e660);

Code was "changed", but the actual problem is still there, as my change request update has documented in detail. I should probably stop writing detailed change requests: Overall, they waste more time than they save.